### DISCUSSION TITLE
Link tooltip opens behind CP navigation

### DISCUSSION THREAD

**Discussion by Joorren** [craftcms/redactor#404] [Ideas]
### Description
When clicking on a link in a redactor field, the tooltip containing the link info will appear behind the Control Panel navigation.

![](https://i.imgur.com/bHNnGai.png)

### Steps to reproduce

1. Add a link to a redactor field
2. Click on the link to open the tooltip

### Additional info

- Craft version: 3.6.15
- PHP version: 8.0
- Database driver & version: MySQL 8
- Plugins & versions: Redactor 2.8.7

**Comment by andris-sevcenko** [comment]
Currently, it's the expected behavior. If it goes above the sidebar, it will go above the top header, too, which will be even uglier. So, can't solve this for now until Craft rearranges things on its end, z-index wise.
